app/forms/waste_carriers_engine/copy_cards_bank_transfer_form.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 module WasteCarriersEngine
-  class CopyCardsBankTransferForm < BankTransferForm
+  class CopyCardsBankTransferForm < BaseForm
```

Comment:
Why are we changing this? I thought the scope of the ticket wasn't including copy cards flow?

Reply:
Ah! Never mind. I got it :)